src/settings/index.ts

```diff
@@ -1,6 +1,7 @@
 import { AceBaseStorageSettings } from 'acebase';
 import { readFileSync } from 'fs';
 import { AceBaseServerEmailSettings } from './email';
+import { Server } from 'http';
```

Comment:
adding to a Server from https must also be allowed

Reply:
unless Server from https inherits from http Server of course!

Reply:
It is inherited actually. From `https.d.ts`:
```ts
    interface Server<
        Request extends typeof http.IncomingMessage = typeof http.IncomingMessage,
        Response extends typeof http.ServerResponse = typeof http.ServerResponse,
    > extends http.Server<Request, Response> {}
```

And just a sanity check. TypeScript doesn't get mad at this:
```ts
import type { Server as HTTPServer } from 'http';
import { createServer } from "https";

const serv = createServer();
func(serv);

function func(server: HTTPServer) {}
```

Reply:
Perfect!